manipulation/exercises/mobile/test_mobile_base_ik.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+import timeout_decorator
+from gradescope_utils.autograder_utils.decorators import weight
+import numpy as np
+from pydrake.all import RotationMatrix, RigidTransform
+
+
+class TestMobileBaseIk(unittest.TestCase):
+    def __init__(self, test_name, notebook_locals):
+        super().__init__(test_name)
+        self.notebook_locals = notebook_locals
+
+    @weight(5)
+    @timeout_decorator.timeout(10.0)
+    def test_mobile_base_ik(self):
+        """Test solve_ik"""
+        # yapf: disable
+        goal_rotation = RotationMatrix([  # noqa
+            [0.9969323810421103, -0.0015776037860101618, -0.07825176544975043],
+            [-0.010615080452914535, -0.9932842569807606, -0.11521156583067553],
+            [-0.07754448849974165, 0.11568878943299686, -0.9902539857545847],
```

Comment:
i'm curious why you have these crazy looking numbers..? (here and in the solns?)

Reply:
I had originally obtained these by teleoperating the robot to some interesting poses and storing them. I've switched to more intuitive poses.